CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Line selection will now expand across wrapped lines
 - The default value for `draw_bold_text_with_bright_colors` is now `false`
 - Mirror OSC query terminators instead of always using BEL
+- Beam, Underline, and Hollow Block cursors line width computation to be more visible
```

Comment:
While `Changed` should not be duplicated in changlog entries, the entry itself should still read as a sentence.

The current version misses a verb, which is why I've originally suggested changing it to `Increased Beam, Underline and Box cursors widths`.

```suggestion
- Increased Beam, Underline, and Hollow Block cursors line widths
```

Reply:
This got addressed right as I was writing my usual nitpick comment:

`s/cursors/cursors'/` -- plural possessive

Reply:
I love it, of course.